### PR TITLE
refactored from redux to context api

### PR DESCRIPTION
**Pull request by Khizar-BM**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -8,38 +8,35 @@ import Board from './components/pages/Board';
...
 
 // Redux
-import { Provider } from 'react-redux';
-import store from './store';
-import { loadUser } from './actions/auth';
 import setAuthToken from './utils/setAuthToken';
 
 import './App.css';
+import {AuthContext} from "./contexts/authStore";
+import BoardStore from "./contexts/boardStore";
```
Comment (omerzn) [inline client/src/App.js:15]: why are these two being imported differently? Uniformity is very important.

File: client/src/components/board/CreateList.js [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
 import CloseIcon from '@material-ui/icons/Close';
+import {BoardContext} from "../../contexts/boardStore";
 
 const CreateList = () => {
-  const [adding, setAdding] = useState(false);
-  const [title, setTitle] = useState('');
-  const dispatch = useDispatch();
+    const [adding, setAdding] = useState(false);
+    const [title, setTitle] = useState('');
+    const {addList} = useContext(BoardContext);
```
Comment (omerzn) [inline client/src/components/board/CreateList.js:10]: another thing i have noticed is that you haven't used an HOC to pass context. why is that?

**Review by omerzn — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (Khizar-BM) [reply client/src/App.js:15]: Both are different. BoardStore is the store wrapper and AuthContext is to be used with useContext

File: client/src/components/board/CreateList.js (hunk 2 above)
Comment (Khizar-BM) [reply client/src/components/board/CreateList.js:10]: I read online that it is not preferred to use HOC now due to the simplicity introduced by UseContext Hook.

File: client/src/App.js (hunk 1 above)
Comment (Khizar-BM) [reply client/src/App.js:15]: the AuthStore (like BoardStore here) is used in index.js

**Pull request merged**